tools/rostest/src/rostest/rostest_main.py

```diff
@@ -147,7 +150,9 @@ def rostestmain():
         parser.error("test file is invalid. Generated failure case result file in %s"%results_file)
         
     try:
-        testCase = rostest.runner.createUnitTest(pkg, test_file)
+        if not options.port:
+            options.port = None
```

Comment:
These to lines are redundant. The options already contain `port` with the value `None` if not specified.